.github/workflows/ci.yml

```diff
@@ -1,60 +1,127 @@
 name: CI
 
 on:
-  pull_request:
   push:
-      branches:
+  pull_request:
+    branches:
       - master
 
 jobs:
   mix_test:
     name: mix test (OTP ${{matrix.otp}} | Elixir ${{matrix.elixir}})
+
     strategy:
       matrix:
         include:
-          - otp: 21.3.8.16
-            elixir: 1.7.4
-          - otp: 21.3.8.16
-            elixir: 1.8.2
-          - otp: 21.3.8.16
-            elixir: 1.9.4
-          - otp: 21.3.8.16
-            elixir: 1.10.4
-          - otp: 23.0.2
-            elixir: 1.10.4
+          - elixir: 1.7.4
+            otp: 21.3.8.16
+
+          - elixir: 1.8.2
+            otp: 21.3.8.16
+
+          - elixir: 1.9.4
+            otp: 21.3.8.16
+
+          - elixir: 1.10.4
+            otp: 21.3.8.16
+
+          - elixir: 1.10.4
+            otp: 23.0.2
+
+          - elixir: 1.11.4
+            otp: 23.3.1
+            lint: lint
+
     runs-on: ubuntu-latest
+
     steps:
-    - uses: actions/checkout@v2
-    - uses: erlef/setup-elixir@v1
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Set up Elixir
+      uses: erlef/setup-elixir@v1
       with:
-        otp-version: ${{matrix.otp}}
-        elixir-version: ${{matrix.elixir}}
-    - name: Install Dependencies
-      run: |
-        mix local.rebar --force
```

Comment:
rebar and hex are updated in `erlef/setup-elixir` action itself, no need to do that again.